RecoTauTag/RecoTau/plugins/PATTauDiscriminationAgainstElectronDeadECAL.cc

```diff
@@ -0,0 +1,80 @@
+
+/** \class PATTauDiscriminationAgainstElectronDeadECAL
+ *
+ * Flag tau candidates reconstructed near dead ECAL channels,
+ * in order to reduce e -> tau fakes not rejected by anti-e MVA discriminator
+ *
+ * Adopted from RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationAgainstElectronDeadECAL.cc
+ * to enable computation of the discriminator on MiniAOD
+ *
+ * The motivation for this flag is this presentation:
+ *   https://indico.cern.ch/getFile.py/access?contribId=0&resId=0&materialId=slides&confId=177223
+ *
+ * \authors Lauri Andreas Wendland,
+ *          Christian Veelken
+ *
+ *
+ *
+ */
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include <FWCore/ParameterSet/interface/ConfigurationDescriptions.h>
+#include <FWCore/ParameterSet/interface/ParameterSetDescription.h>
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/TauReco/interface/PFTauFwd.h"
+#include "DataFormats/TauReco/interface/PFTauDiscriminator.h"
+#include "RecoTauTag/RecoTau/interface/TauDiscriminationProducerBase.h"
+#include "RecoTauTag/RecoTau/interface/AntiElectronDeadECAL.h"
+
+class PATTauDiscriminationAgainstElectronDeadECAL : public PATTauDiscriminationProducerBase {
```

Comment:
considering that most of the code in PFRecoTauDiscriminationAgainstElectronDeadECAL and this file is now rather "boilerplate", the need for a template is not very strong. Still, it would be better if the implementation comes out rather straightforward.